### PR TITLE
Functions

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
code pr
https://github.com/vesoft-inc/nebula-common/pull/314

### CODE REVIEW

File: docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md [hunk 1]
```diff
@@ -45,17 +45,17 @@ nebula> INSERT VERTEX date(p1, p2, p3) VALUES "Date1":(date("2017-03-04"), time(
 Create a tag named school.
 
 ```ngql
-nebula> CREATE TAG school(name string , create_time timestamp);
+nebula> CREATE TAG school(name string , found_time timestamp);
 ```
 
 Insert a vertex named "stanford" with the foundation date `"1885-10-01 08:00:00"`.
```
Comment (laura-ding) [inline docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md:51]: "1885-10-0T08:00:00"

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md [hunk 2]
```diff
@@ -4,32 +4,33 @@ Nebula Graph supports the following built-in math functions:
...
+double acos(double x) | Returns inverse cosine of the argument. |
+double tan(double x) | Returns tangent of the argument. |
+double atan(double x) | Returns inverse tangent the argument. |
+int rand32() | Returns a random 32 bit integer. |
+int rand32(int max) | Returns a random 32 bit integer in [0, max).  |
+int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
+int rand64() | Returns a random 64 bit integer. |
+int rand64(int max) | Returns a random 64 bit integer in [0, max). |
+int rand64(int min, int max) | Returns a random 64 bit integer in [min, max).|
+collect() | Puts all the collected values to a list.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:32]: better to give examples from cypher collect()/collect_set()/map()/range()

File: docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md [hunk 3]
```diff
@@ -71,10 +71,12 @@ If you are reading this topic with the questions listed below, click them to jum
 4. Connect to Nebula Graph.
 
     ```bash
+    # Start a Nebula Graph service in a Docker container
```
Comment (oldLady344) [inline docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md:74]: Nebula Console, not Nebula Graph

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:32]: I will add later

**Review by randomJoe211 — APPROVED** [review]
💯

**Pull request merged**